### PR TITLE
[lcms] update to version 2.14

### PR DESCRIPTION
**Pull request by 0xC0000054**
- #### What does your PR fix?
  Updates lcms to version 2.14

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?
  all, No

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?
   Yes

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?
  Yes

### CODE REVIEW

File: ports/lcms/vcpkg.json [hunk 1]
```diff
@@ -1,7 +1,6 @@
 {
   "name": "lcms",
-  "version": "2.12",
-  "port-version": 4,
+  "version": "2.14",
   "description": "Little CMS.",
```
Comment (JonLiu1993) [inline ports/lcms/vcpkg.json:4]:
```suggestion
  "description": "Little CMS.",
  "license": "MIT",
```

Comment (JonLiu1993) [reply ports/lcms/vcpkg.json:4]: https://github.com/mm2/Little-CMS/blob/master/COPYING

**Review by JonLiu1993 — CHANGES_REQUESTED** [review]

File: ports/lcms/vcpkg.json (hunk 1 above)
Comment (JonLiu1993) [reply ports/lcms/vcpkg.json:4]: @0xC0000054 , Thanks for your pr, please add license here.

Comment (0xC0000054) [reply ports/lcms/vcpkg.json:4]: Added in commit https://github.com/microsoft/vcpkg/commit/9ae28988cf5ab4c73d3e155ba35bdf5c63547e58

**Review by JonLiu1993 — DISMISSED** [review]

**Comment by JonLiu1993** [comment]
Please run command "./vcpkg x-add-version lcms --overwrite-version" and commit again.

**Review by JonLiu1993 — APPROVED** [review]

**Comment by JonLiu1993** [comment]
Duplicate of #27593

**Pull request closed without merging**